concepts/random/introduction.md

```diff
@@ -0,0 +1,125 @@
+# Introduction
```

Comment:
I decided to keep the intro and about almost exactly the same, because pretty much everything is needed for solving the exercise.
The only change I did is remove all the links, because they are considered distracting.